File: calculadora-csharp-mvc-reflection/Calculadora/DTO/RequestDTO.cs
namespace Calculadora.DTO
{
    public class RequestDTO
    {
        public string Opcao { get; set; }
        public int Valor1 { get; set; }
        public int Valor2 { get; set; }

        public RequestDTO(string opcao, int valor1, int valor2)
        {
            Opcao = opcao;
            Valor1 = valor1;
            Valor2 = valor2;
        }
    }
}

File: calculadora-java-reflection/Calculadora/src/br/ifes/projetosistemas/calculadora/dto/RequestDTO.java
package br.ifes.projetosistemas.calculadora.dto;

public class RequestDTO {
    private String operacao;
    private int valor1;
    private int valor2;

    public RequestDTO(String operacao, int valor1, int valor2) {
        this.operacao = operacao;
        this.valor1 = valor1;
        this.valor2 = valor2;
    }

    public String getOperacao() {
        return operacao;
    }

    public void setOperacao(String operacao) {
        this.operacao = operacao;
    }

    public int getValor1() {
        return valor1;
    }

    public void setValor1(int valor1) {
        this.valor1 = valor1;
    }

    public int getValor2() {
        return valor2;
    }

    public void setValor2(int valor2) {
        this.valor2 = valor2;
    }
}
